src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -0,0 +1,200 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size, screenSize } from './theme';
+import { P, ArticleH3, H3 } from './text';
+import Input from './input';
+import Button from './button';
+import Select from './select';
+import Checkbox from '@leafygreen-ui/checkbox';
+import SectionHeader from './section-header';
+import SuccessState from './success-state';
+
+const StyledButton = styled(Button)`
+    display: flex;
+    margin: ${size.large} auto;
+`;
+
+const StyledSelect = styled(Select)`
+    color: ${colorMap.greyLightTwo};
+`;
+
+const ErrorMessage = styled(P)`
+    color: ${colorMap.salmon};
+`;
+
+const StyledSectionText = styled(ArticleH3)`
+    color: ${colorMap.greyLightTwo};
+    font-family: akzidenz;
+    font-weight: normal;
+`;
+
+const InstructorSection = styled('div')`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, 335px);
+    grid-row-gap: ${size.mediumLarge};
+    justify-content: space-between;
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToMedium} {
+        display: block;
+    }
+`;
+
+const InstitutionSection = styled('div')`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, 335px);
+    grid-row-gap: ${size.mediumLarge};
+    justify-content: space-between;
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToMedium} {
+        display: block;
+    }
+`;
+
+const StyledForm = styled('form')`
```

Comment:
We'll likely want to apply max-width and width to the general page layout as well, but we can do that separately and then remove here later.